docs/layers.md

```diff
@@ -1595,34 +1595,34 @@ The `hgv` property indicates general truck heavy goods vehicle truck access, val
-* `day_week_year` - range of days of the week of the year, for example, from Friday of 12th week until Friday of 36th week - `day_week_year Wed12-Fri36`
-* `week_of_month` - range of weeks of the month, for example, from 1st until 4th week of month `week_of_month 1-4`
-* `month_of_year` - range of months of the year, for example, `month_of_year Jan-Jul`
-* `day_month_year` - range of days of months of the year, for example, `day_month_year Jan1-Jul31`
-* `day_week_month_year` - range of days of weeks of months of the year, for example, from 1st Sunday of July until 1 Monday of December `day_week_month_year JulSun1-DecMon1`
-* `externally_specified` - externally specified string value as a raw text, for example for the holiday season, `externally_specified Easter`
-* `dusk_to_dawn` - during nighttime
-* `dawn_to_dusk` - during daytime
-* `start_time` - seconds since midnight when the restriction starts, for example,
-  `start_time25200`; if specified, `end_time` must also be specified
-* `end_time` - seconds since mighnight when the restriction ends, for example,
-  `end_time72000`; if specified, `start_time` must also be specified
-* `excluded` - if the restriction is excluded, then the time period specified is the period during which the restriction does not apply,
-for example, `excluded days_of_week Sat,Sun|start_time25200end_time72000`
+Property specifies date and time period during which the condition applies. Value is a string in the Time Domain format. Time Domain is part of the GDF(Geographic Data Files) specification which is an ISO standard. Current standard is GDF 5.1 which is ISO 20524-1:2020.
+
+A basic Time Domain is the combination of a Starting Date and a Time Duration with the following notation: [(Starting Date) {Time duration}]. For example, `[(M5d1){d1}]` means: Starting Date: any year, month 5 (May), day 1st, at 0:00 a.m. Duration: 1 complete day (i.e., 24 hours or 1440 minutes).
+
+Starting Dates and time intervals are defined by means of a set of graphical symbols allowing the description of years, months, weeks, days, and so on down to the smallest time unit, which is the second. The symbols have to be organized in a sequential order starting with the longest time unit. Attached to a starting date, the interval constitutes a basic Time Domain. Without a starting date, it just indicates a duration. If the very first time type code in duration section is preceded by a minus sign, it means that the duration is counted in the reverse order. The following list describes starting date and time interval unit types:
+* `ynn` - Year. Defines either a particular nnnn year in a starting date; or a duration of nn years. For example, [(y1991M11d14h5m30s19){y1}] means from 14 November 1991, 5:30:19 a.m. to 14 November 1992, 5:30:19 a.m. If there is no identical calendar date in the year in question, which occurs only for 29 February, “plus 1 year” leads to 28 February of the following year. Notice that {y1 = {M12}
```

Comment:
Nit 1: add whitespace newline return before starting a list.

Nit 2: any example code should be quoted, like `[(y1991M11d14h5m30s19){y1}]`

Nit 3: `{y1 = {M12}` to `{y1} = {M12}` (add missing `}` brace?

Reply:
fixed